docs/remove.md

```diff
@@ -227,3 +229,12 @@ Create a "base" subset by removing external axioms (alternatively, use `filter -
       --select annotation-properties \
       --signature true \
       --output results/filter_annotations.owl
+
+Create a "base" subset that axiom annotations related with the two properties are removed:
```

Comment:
Create a "base" subset in which axiom annotations involving IAO:0000117 and IAO:0000119 are removed:

Reply:
Fixed